### PR TITLE
implemented registration page

### PR DESCRIPTION
**Pull request by syedsaifshah**
1 create registration form
2 add editview icons

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -4,13 +4,15 @@
...
 
+
     <ImageView
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:tint="@color/black_faded"
         android:scaleType="centerCrop"
         android:src="@drawable/img_1" />
 
+
```
Comment (sayedmafhh) [inline app/src/main/res/layout/activity_main.xml]: please make code reformat

File: app/src/main/res/layout/activity_registration.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        android:layout_height="match_parent"
+        android:scaleType="centerCrop"
+        android:src="@drawable/img_1"
+        android:tint="@color/black_faded" />
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_centerInParent="true"
+        android:layout_margin="20dp">
```
Comment (sayedmafhh) [inline app/src/main/res/layout/activity_registration.xml]: define dp in dimen folder

File: app/src/main/res/layout/activity_registration.xml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+        <EditText
+            android:id="@+id/edt_name"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/edt_ferrari_logo"
+            android:drawableStart="@drawable/ic_name_tint"
+            android:drawablePadding="20dp"
+            android:layout_marginTop="40dp"
+            android:layout_marginBottom="20dp"
```
Comment (sayedmafhh) [inline app/src/main/res/layout/activity_registration.xml]: define dp in dimen folder

File: app/src/main/res/layout/activity_registration.xml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+            android:hint="@string/hint_name"
+            android:textSize="20sp" />
+
+        <EditText
+            android:id="@+id/edt_username"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/edt_name"
+            android:layout_marginBottom="20dp"
```
Comment (sayedmafhh) [inline app/src/main/res/layout/activity_registration.xml]: define dp in dimen folder

File: app/src/main/res/layout/activity_registration.xml [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+            android:drawablePadding="20dp"
+            android:hint="@string/hint_username"
+            android:textSize="20sp" />
+
+        <EditText
+            android:id="@+id/edt_password"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:drawableStart="@drawable/ic_password_tint"
+            android:drawablePadding="20dp"
```
Comment (sayedmafhh) [inline app/src/main/res/layout/activity_registration.xml]: define dp in dimen folder

File: app/src/main/res/layout/activity_registration.xml (hunk 4 above)
Comment (syedsaifshah) [reply app/src/main/res/layout/activity_registration.xml]: ok

**Pull request merged**